doc/whats_new.rst

```diff
@@ -11,7 +11,7 @@ New
 
 * Use :func:`nistats.reporting.make_glm_report` to easily generate HTML reports from fitted first and second level models and contrasts.
 * New dataset fetcher, :func:`nistats.datasets.fetch_language_localizer_demo_dataset` , BIDS 1.2 compatible.
-* New example showcasing the use of a GLM to get beta maps for decoding experiments (aka beta-regression) in :func:`examples/05_complete_examples/plot_haxby_block_classification.py`
+* New example showcasing the use of a GLM to get beta maps for decoding experiments (aka beta-regression).
```

Comment:
We should add the link. I'll find another way.

Reply:
Glad to learn how you do that !